Formula/freetype.rb

```diff
@@ -28,6 +28,15 @@ class Freetype < Formula
   uses_from_macos "bzip2"
   uses_from_macos "zlib"
 
+  fails_with :clang do
+    build 1000
```

Comment:
```suggestion
    build 1100
```
`freetype` build fails on Mojave too, but succeeds on Catalina, so might as well `fails_with` on Mojave clang.